app/views/reports/_team_score.html.erb

```diff
@@ -1,6 +1,14 @@
-<% if !@assignment.vary_by_round %>
+<% unless @assignment.varying_rubrics_by_round? %>
     <!--Score awarded / Avg. score-->
     <div>
+    <% if @review_scores[reviewer_id][team_id] && @review_scores[reviewer_id][team_id] != -1 %>
```

Comment:
move the `@review_scores[reviewer_id][team_id] != -1` into the next unless and use 
`@review_scores[reviewer_id][team_id] == -1`